packages/design-system-react-native/package.json

```diff
@@ -39,22 +34,47 @@
     "build:docs": "typedoc",
     "changelog:update": "../../scripts/update-changelog.sh @metamask/design-system-react-native",
     "changelog:validate": "../../scripts/validate-changelog.sh @metamask/design-system-react-native",
+    "postinstall": "cd ios && pod install || exit 0",
     "publish:preview": "yarn npm publish --tag preview",
+    "run-android": "react-native run-android",
+    "run-ios": "react-native run-ios --scheme MetamaskDesignSystemApp",
     "since-latest-release": "../../scripts/since-latest-release.sh",
-    "test": "NODE_OPTIONS=--experimental-vm-modules jest --reporters=jest-silent-reporter",
-    "test:clean": "NODE_OPTIONS=--experimental-vm-modules jest --clearCache",
-    "test:verbose": "NODE_OPTIONS=--experimental-vm-modules jest --verbose",
-    "test:watch": "NODE_OPTIONS=--experimental-vm-modules jest --watch"
+    "start": "react-native start",
+    "test": "jest --reporters=jest-silent-reporter",
+    "test:clean": "jest --clearCache",
+    "test:verbose": "jest --verbose",
+    "test:watch": "jest --watch"
+  },
+  "dependencies": {
+    "react": "18.2.0",
+    "react-native": "0.72.0"
   },
```

Comment:
should be in devDependencies and peerDependencies (in peer it should not be a strict version e.g. 18.0.0)